spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,18 @@
+ {
+   "srcDir": "public/assets",
+   "srcFiles": [
+     "govuk-admin-template-*.js",
+     "application-*.js"
+   ],
+   "cssFiles": [
+     "application-*.css"
+   ],
```

Comment:
These don't seem to match the files that were specified in the previous jasmine config: 
 https://github.com/alphagov/collections-publisher/blob/301003f7bbfe437e7dda5413419c4dcd050b3b11/spec/javascripts/support/jasmine.yml#L13-L28

I imagine we want:

```
   "srcFiles": [
     "govuk-admin-template-*.js",
     "admin_layout-*.js"
   ],
   "cssFiles": [
     "admin_layout-*.css"
   ],
```

for consistency.

Reply:
Thank you @kevindew, it makes sense!